IntegrationTests/tests_01_http/defines.sh

```diff
@@ -149,6 +155,7 @@ function stop_server() {
         if ! kill -0 "$token_pid" 2> /dev/null; then # ignore-unacceptable-language
             break # good, dead
         fi
+        # shellcheck disable=SC2009
```

Comment:
Didn't fancy `pgrep` or just minimising churn?

Reply:
The `kill -0 "$token_pid"` does the right thing. IIRC, the bad-looking `ps auxw | grep "$token_pid" || true` is for actual human output. The idea is: We check (correctly) with `kill -0` is a pid is dead. But if it isn't, then we present the user with the well-known `ps axuw` output so they get to see what's up.

I'm happy to also delete `ps auxw | grep "$token_pid" || true` without replacement. This hasn't failed in 6 years and if it does we can re-add debugging.

Reply:
Yes minimising churn since this didn't seem required but more a suggestion